pages/_app.js

```diff
@@ -1,7 +1,21 @@
-import '../styles/globals.css'
+import '../styles/globals.css';
+import '../Bootstrap/css/bootstrap.min.css';
```

Comment:
We don't want to import all of bootstrap!

Reply:
https://getbootstrap.com/docs/3.4/customize/#colors, I created customized bootstrap and checked whatever is needed, and thus imported the whole thing, cause this is all we need.